docs/book/how-to/build-pipelines/tag-your-pipeline-runs.md

```diff
@@ -0,0 +1,26 @@
+# Tag your pipeline runs
+
+You can specify tags for your pipeline runs in the following places
+
+- in the configuration file:
+```yaml
+# config.yaml
+tags:
+  - tag_in_config_file
+```
+
+- in code on the `@pipeline` decorator or using the `with_options` method:
+```python
+@pipeline(tags=["tag_on_decorator"])
+def my_pipeline():
+  ...
+
+my_pipeline = my_pipeline.with_options(tags=["tag_on_with_options"])
+```
+
+If you now run this pipeline, tags from all these places will be merged and applied to the pipeline run.
```

Comment:
do you want to link to this page that talks about this: https://docs.zenml.io/how-to/use-configuration-files/configuration-hierarchy

Reply:
I think this page is more step/pipeline level, which does not exist for tags. Additionally, tags are actually a special case where we merge lists, which is not happening in any other configuration, so I'd leave it out personally. WDYT?

Reply:
ok